packages/browser-agent-custom/index.js

```diff
@@ -0,0 +1,57 @@
+// loader files
+import { getEnabledFeatures } from '@newrelic/browser-agent-core/src/loader/enabled-features'
+import { configure } from '@newrelic/browser-agent-core/src/loader/configure'
+import { getFeatureDependencyNames } from '@newrelic/browser-agent-core/src/loader/featureDependencies'
+import { featurePriority } from '@newrelic/browser-agent-core/src/loader/features'
+// common files
+import { Aggregator } from '@newrelic/browser-agent-core/src/common/aggregate/aggregator'
+import { gosNREUMInitializedAgents } from '@newrelic/browser-agent-core/src/common/window/nreum'
+import { generateRandomHexString } from '@newrelic/browser-agent-core/src/common/ids/unique-id'
+import { getConfiguration, getInfo, getLoaderConfig, getRuntime } from '@newrelic/browser-agent-core/src/common/config/config'
+
+
+export class BrowserAgent {
```

Comment:
Can we move this into a `src` directory to help keep a clean distinction between package configuration files and actual source code?